### DISCUSSION TITLE
How to use a Private CA in WSA?

### DISCUSSION THREAD

**Discussion by MarkLFT** [microsoft/WSA#286] [General]
I have developed an internal app that uses a certificate signed with our own private CA.  When using an emulator or physical device, I can install the CA onto the device for our app to run.   How can I install a private Ca for WSA to use?

**Comment by nieubank** [comment]
Does cert installer not work when the certificate is pushed to /sdcard from adb or placed in documents/downloads?

e.g. adb shell am start -n com.android.certinstaller/.CertInstallerMain -a android.intent.action.VIEW -t application/x-x509-ca-cert -d file:///sdcard/test.cer
